owl/EASE-SAY.owl

```diff
@@ -123,6 +331,75 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ESTSchemaTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ESTSchemaTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#ImageSchemaTheory"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#PhysicalObjectRole"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:cardinality>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The ESTSchema constitutes the most basic schema that describes the existance of an entity, thereby having one Role, i.e. what entity is asserted to exist.
```

Comment:
I do not get the part "thereby having one Role". Who has one role, and why? That is not clear to me.

Reply:
I'm not sure where the confusion comes from. It's pretty clear the schema being defined has a single role.

Reply:
" that describes the existance of an entity, thereby having one Role" 
so how does describing the existence implies that there is one role? I do not get it. Please rephrase.

There is also a typo, it is written **existence**

Reply:
There is one role because the schema refers to the object existing. There is no need for any other role.

Reply:
The "existance" typo has already been resolved by my latest commit yesterday; as github shows, this convo is on outdated code.